docs/projects/adv7511/index.rst

```diff
@@ -0,0 +1,241 @@
+.. _adv7511:
+
+ADV7511 HDL project
+===============================================================================
+
+Oveview
+-------------------------------------------------------------------------------
+
+The :adi:`ADV7511` is a 225 MHz High-Definition Multimedia Interface (HDMI®) transmitter,
+which is ideal for home entertainment products including DVD players/recorders, digital
+set top boxes, A/V receivers, gaming consoles, and PCs.
+
+The digital video interface contains an HDMI 1.4- and a DVI 1.0-compatible transmitter,
+and supports all HDTV formats (including 1080p with 12-bit Deep Color). The :adi:`ADV7511`
+supports the HDMI 1.4-specific features, HEAC (ARC), and 3D video. In addition to these features,
+the :adi:`ADV7511` supports x.v.Color™, high bit rate audio, and programmable AVI InfoFrames.
+With the inclusion of HDCP, the :adi:`ADV7511` allows the secure transmission of protected content
+as specified by the HDCP 1.4 protocol.
+
+The :adi:`ADV7511` supports both S/PDIF and 8-channel I2S audio. Its high fidelity 8-channel I2S
+can transmit either stereo or 7.1 surround audio up to 768 kHz. The S/PDIF can carry compressed
+audio including Dolby® Digital, DTS®, and THX®. Fabricated in an advanced CMOS process, the :adi:`ADV7511`
+is provided in a 100-lead LQFP surface-mount plastic package and is specified over the 0°C to
++70°C temperature range.
+
+Supported boards
+-------------------------------------------------------------------------------
+
+-  :adi:`ADV7511`
```

Comment:
```suggestion
- :adi:`EVAL-ADV7842`
```
as in the [readme ](https://github.com/analogdevicesinc/hdl/blob/main/projects/adv7511/Readme.md)

Reply:
The eval-kit from the [readme](https://github.com/analogdevicesinc/hdl/blob/main/projects/adv7511/Readme.md) of the project's directory is only an evaluation kit of the ADV7511 (in this case, the FPGA-based ADV7511 projects exposes the use of ADV7511 on different FPGA SoC Evaluation kits from Xilinx - like ZC706/ZC702/Zed).